### ISSUE TITLE
neutrino tau unused

### ISSUE THREAD

**Issue by zingale** [starkiller-astro/Microphysics#870]
it looks like the photo neutrino losses want T > 1.e7 otherwise tau is not defined:
```
/ccs/home/zingale/Microphysics/neutrinos/sneut5.H:705:16: warning: variable 'tau' is used uninitialized whenever 'if' condition is false [-Wsometimes-
uninitialized]
    } else if (temp >= 1.0e9_rt) {
```

**Comment by zingale** [comment]
this is true of a lot of the variables in that block.

**Comment by zingale** [comment]
the expression comes from here:
https://articles.adsabs.harvard.edu/pdf/1996ApJS..102..411I

that only provides the photo-neutrino losses for 1.e7 < T < 1.e10.  So when T < 1.e7, we have an undefined state.  Since we are not likely burning at this temperature anyway, we should probably just zero out the photo neutrinos.

**Comment by maxpkatz** [comment]
That already happens in the current code: https://github.com/starkiller-astro/Microphysics/blob/main/neutrinos/sneut5.H#L331-L337

**Comment by zingale** [comment]
ah okay.  The compiler is just not smart enough to see this and gives warnings that things might be unused.

**Comment by zingale** [comment]
the warnings were fixed in #873
